src/main/java/org/jitsi/impl/protocol/xmpp/XmppProviderFactory.java

```diff
@@ -37,7 +37,7 @@
      * @param bundleContext the bundle context reference of the service
      * @param protocolName  the name of the protocol
      */
-    protected XmppProviderFactory(
+    public XmppProviderFactory(
```

Comment:
Is this visibility change temporary (while refactoring things, maybe across other PRs), or fix?  Just wondering if we should comment on it being temporary so others don't use it while the bigger refactoring is taking place.

Reply:
It's not temporary, there's no reason this should not be public (the latest version is a simple one-method interface)